code/datums/gamemode/factions/bloodcult/bloodcult.dm

```diff
@@ -24,7 +24,7 @@ var/veil_thickness = CULT_PROLOGUE
 	for (var/obj/structure/cult/spire/S in cult_spires)
 		S.upgrade()
 
-	for (var/obj/effect/rune/R in rune_list)
+	for (var/obj/effect/rune/R in global_runesets["blood_cult"].rune_list)
```

Comment:
Erf
This will compile if there is a var named `rune_list` on any datum in the world, rather than on the object type of `global_runesets["blood_cult"]`
I would personally typecast it for better compile safety

Reply:
Done.